soh/src/code/z_play.c

```diff
@@ -1408,7 +1408,7 @@ void Gameplay_Draw(GlobalContext* globalCtx) {
                     OVERLAY_DISP = sp70;
                     globalCtx->unk_121C7 = 2;
                     SREG(33) |= 1;
-                } else {
+                } else if (R_PAUSE_MENU_MODE != 3) {
```

Comment:
is there an enum value you can take instead of the magic number `3`? if not LGTM.

Reply:
looks like no enum. :shipit: